Misc/NEWS.d/next/Tests/2020-12-01-15-51-19.bpo-31904.iwetj4.rst

```diff
@@ -0,0 +1,5 @@
+Create fake home directory in temporary directory in test_security(),
+test_file_not_found_in_home() and test_home_not_set() of test_netrc.py.
+Previously fake home was created under the current directory. However,
+if they are not run up by reression test, the current directory could
+be not under /tmp, it probably has permission problem.
```

Comment:
I don't think that the NEWS entry is useful for end users. Or write something shorter like:

"Fix test_netrc on VxWorks: create temporary directories using temp_cwd()."

Reply:
Fixed.